paimon-web-common/src/main/java/org/apache/paimon/web/common/result/SubmitResult.java

```diff
@@ -0,0 +1,129 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.paimon.web.common.result;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+/** This class represents the result of a job. Including flink job and spark Job. */
+public class SubmitResult implements Serializable {
+
+    private static final long serialVersionUID = 1L;
+
+    private final String submitId;
```

Comment:
What's the purpose of definition for `submitId`?

Reply:
If it is sql gateway, submitId will store operationHandle in order to obtain the results, and others will store a uuid.